src/picker/rofimoji.py

```diff
@@ -448,26 +452,28 @@ def append_to_favorites_file(self, characters: str) -> None:
             file.write(characters + '\n')
 
     def execute_action(self, characters: str) -> None:
-        if self.args.action == Action.TYPE:
-            self.typer.type_characters(characters, self.active_window)
-        elif self.args.action == Action.COPY:
-            self.clipboarder.copy_characters_to_clipboard(characters)
-        elif self.args.action == Action.CLIPBOARD:
-            self.clipboarder.copy_paste_characters(characters, self.active_window, self.typer)
-        elif self.args.action == Action.UNICODE:
-            self.typer.type_characters(self.get_codepoints(characters), self.active_window)
-        elif self.args.action == Action.COPY_UNICODE:
-            self.clipboarder.copy_characters_to_clipboard(self.get_codepoints(characters))
-        elif self.args.action == Action.STDOUT:
-            print(characters)
+        for action in self.args.actions:
+            if action == Action.TYPE:
+                self.typer.type_characters(characters, self.active_window)
+            elif action == Action.COPY:
+                self.clipboarder.copy_characters_to_clipboard(characters)
+            elif action == Action.CLIPBOARD:
+                self.clipboarder.copy_paste_characters(characters, self.active_window, self.typer)
+            elif action == Action.UNICODE:
+                self.typer.type_characters(self.get_codepoints(characters), self.active_window)
+            elif action == Action.COPY_UNICODE:
+                self.clipboarder.copy_characters_to_clipboard(self.get_codepoints(characters))
+            elif action == Action.STDOUT:
+                print(characters)
 
     def save_selection_to_cache(self, characters: str, processed_characters: str) -> None:
         with cache_file_location.open('w+') as file:
-            file.write(f'{self.args.action.value}\n{characters}\n{processed_characters}')
+            actions = ",".join(action.val for action in self.args.actions)
+            file.write(f'{actions}\n{characters}\n{processed_characters}')
```

Comment:
I think I got this update right (and the one below), but I honestly wasn't sure how to trigger this caching mechanism to test it. If you can give me a pointer on when this is used, I'll double-check that it works.

Reply:
This whole thing is for the rofi mode, i.e. calling `rofi -modi "emoji:<path to rofimoji.py>" -show emoji`.